doc/api/deprecations.md

```diff
@@ -95,12 +95,16 @@ methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
 <a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
+### DEP0007: Replace cluster worker.suicide with worker.exitedAfterDisconnect
 
-Type: Runtime
+Type: End-of-Life
```

Comment:
@addaleax We both wrote our comments within a minute, I did not see your comment until I had written mine, I was not responding to you ;)